### ISSUE TITLE
Filters don't work on Get-GAGroupMember

### ISSUE THREAD

**Issue by mattfippard** [squid808/gShell#7]
If I run Get-GAGroupMember -GroupName mygroup -Owners or with the -Managers or -Members filters all group members are returned

**Comment by squid808** [comment]
Now that the authentication hell is more or less [hopefully] done, I'll look in to this more for the next release.

**Comment by Fetch04** [comment]
I believe this is probably because of the object type that get-GAGroupMember returns.  

```
PS H:\> Get-GAGroupMember -GroupName "my-group@mydomain.com" | gm | select typename -First 1

TypeName                                                                                                                                                                                       
--------                                                                                                                                                                                       
System.Collections.Generic.List`1[[Google.Apis.Admin.Directory.directory_v1.Data.Member, Google.Apis.Admin.Directory.directory_v1, Version=1.9.2.55, Culture=neutral, PublicKeyToken=4b01fa6...
```

this isn't probably what you were wanting it to return as.  Although it returns correct data:

```
Email : user@mydomain.com
ETag  : "iwpzoDgSq9BJw-XzORg0bILYPVc/VcAszASh6HTLHASD2435P7YCg"
Id    : 12435765265656752
Kind  : admin#directory#member
Role  : OWNER
Type  : USER
```
You can't select using Email or Role or whatever using select-object.

**Comment by Fetch04** [comment]
Another note: if you save the output of get-gagroupmembers to a variable, it uses a correct type:


```
PS H:\> $GoogleGroupMembers | gm


   TypeName: Google.Apis.Admin.Directory.directory_v1.Data.Member

Name        MemberType Definition                    
----        ---------- ----------                    
Equals      Method     bool Equals(System.Object obj)
GetHashCode Method     int GetHashCode()             
GetType     Method     type GetType()                
ToString    Method     string ToString()             
Email       Property   string Email {get;set;}       
ETag        Property   string ETag {get;set;}        
Id          Property   string Id {get;set;}          
Kind        Property   string Kind {get;set;}        
Role        Property   string Role {get;set;}        
Type        Property   string Type {get;set;}        

```

**Comment by squid808** [comment]
Fixed for release 0.0.9.0, thank you! Closing, please test when 0.0.9.0 is released shortly.

@Fetch04 If I understand what you are saying, we definitely do want to return a collection of ```Google.Apis.Admin.Directory.directory_v1.Data.Member``` since this is the class that represents any and all group members. From there, it's just a matter of what the Role property is filled out with, but owners, members and managers are _all_ group members.

**Comment by mattfippard** [comment]
Seems to be working fine in the new version.  Fantastic work on this, saves a lot of time when I can script something that would normally take hours in the web interface!
